lib/healthchecks/healthchecks.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module HealthChecks
+  class QueueDeadSetCheck < OkComputer::Check
+    def check
+      ds = Sidekiq::DeadSet.new
+      size = ds.size
+      mark_failure if size.positive?
+      mark_message "There are #{size} messages in the DeadSet Queue"
+    end
+  end
+
+  class QueueLatencyCheck < OkComputer::Check
+    attr_accessor :threshold
+
+    def initialize(threshold = 30)
+      self.threshold = Integer(threshold)
```

Comment:
Ruby style tip, there are two ways to convert a string to an integer `Integer()` and `.to_i` that behave slightly differently. `nil.to_i #=> 0` while `Integer(nil) #=> raises an exception`

You very, very rarely see `Integer()` used in the wild unless you specifically want that exception raising feature. Which in this case, maybe you do? Or maybe it's an OK Computer idiom? Barring either of those, `.to_i` is much more common to see in Ruby code. Anyways, that just caught my eye